packages/gatsby/src/schema/graphql-engine/bundle-webpack.ts

```diff
@@ -70,6 +70,21 @@ export async function createGraphqlEngineBundle(
     ],
     module: {
       rules: [
+        {
+          test: require.resolve(`lmdb`),
+          parser: { amd: false },
+          use: [
+            {
+              loader: require.resolve(`@vercel/webpack-asset-relocator-loader`),
+              options: {
+                outputAssetBase: `assets`,
+              },
+            },
+            {
+              loader: require.resolve(`./lmdb-bundling-patch`),
+            },
```

Comment:
brittle string replacements to allow binaries and `dict.txt` to be bundled - ideally we can figure out adjustments upstream to allow for bundling without messy patches as what is happening right now